ppr-ui/src/utils/feature-flags.ts

```diff
@@ -9,8 +9,11 @@ export const defaultFlagSet: LDFlagSet = {
   'bcregistry-ui-mhr-enabled': false,
   'search-registration-number': false,
   'search-serial-number': false,
-  'mhr-ui-enabled': false,
-  'mhr-registration-enabled': false,
+  'mhr-ui-enabled': false, // Enables MHR search options
+  'mhr-registration-enabled': false, // Enables MHR table tab
+  'mhr-transfer-enabled': false, // Enables changes to base MHR Home Owners within the MHR Information flow
```

Comment:
So to properly enable transfers we need to enable **mhr ui** first + **mhr transfers**, right?